ngmix/joint_prior.py

```diff
@@ -619,7 +619,7 @@ def get_lnprob_array(self, pars):
 
         return lnp
 
-    def sample(self, n=None):
+    def sample(self, nrand=None):
```

Comment:
docstrings

Reply:
Why is this reolved? The docstring still says `n` and not `nrand`.

Reply:
Yeah, it says `nrand: ` now